mix.exs

```diff
@@ -43,6 +43,7 @@ defmodule Rihanna.MixProject do
   defp deps do
     [
       {:postgrex, ">= 0.13.3"},
+      {:telemetry, "~> 0.4"},
```

Comment:
Do we really want to force people to include telemetry?

Reply:
Telemetry was already in the lockfile, so we wouldn't be adding a dependency. We are going from 0.2 to 0.4, which may be a forced upgrade. I can try reducing the version requirement as I think the interface may be the same.

Reply:
Yeah, ecto_sql already requires telemetry: https://github.com/elixir-ecto/ecto_sql/blob/master/mix.exs#L48